### ISSUE TITLE
how to use the Integration ,when i add navidrome is not thing to do..

### ISSUE THREAD

**Issue by ray5378** [tiorac/ha-subsonic#1]

**Comment by tiorac** [comment]
In this initial version, we only listed the media on HA. Like Radio Browser, but soon I'll implement it as a device to obtain some status. I still need to research a bit about Navidrome's Jukebox to see if it will be possible to control it.

**Comment by mcrypt** [comment]
This is a great project! Looking forward to seeing it come to fruition! 👍🏻

**Comment by ray5378** [comment]
Yes, they can now be seen in the media library, but it indicates that the music is not compatible. We hope it can start up normally so that our navidome server can be perfectly integrated into homeassistant

**Comment by Jypy** [comment]
Thanks for the integration, it looks very promising!
